templates/protocol_switch_service/Ballerina.toml

```diff
@@ -0,0 +1,26 @@
+[project]
+org-name= "wso2"
+version= "1.0.0"
+license= ["Apache-2.0"]
+authors= ["WSO2"]
+keywords= ["ballerina", "integrator"]
+repository= "https://github.com/wso2/ballerina-integrator"
+templates= ["protocol_switch_service"]
+
+[dependencies]
+"wso2/jms"= {path = "/home/thishani/ballerina-modules/module-jms/target/balo/jms-2019r3-java8-0.6.0.balo"}
```

Comment:
Can we give a relative path here? Or we could let it be used as a module pulled from Ballerina Central without pointing the balo?

Reply:
No need of this, hence removed.